### PR TITLE
✨Add Export Functionality On Inspect-Cor…

### LINKED ISSUE #1170: ✨Export diagram from correlation
#### Describe your issue

It would be nice to export a diagram from a specific correlation on the inspect-correlation page.

#### Possible solution

- use the diagram export service

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

### PR DESCRIPTION
**Pull request by alexanderkasten**
**Changes:**

1. Add export functionality to the inspect-correlation view. 

**Issues:**

Closes #1170 

PR: #1222 

---

See the [examples](#example) for inspiration.

## How can others test the changes?

> Describe how others can test your changes (you can remove this section for typo fixes etc.)

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.).
- [x] I've rebased the `develop` branch with my branch before finishing this PR.
- [x] I've **summarized all changes** in a list above.
- [x] I've mentioned all **PRs, which relate to this one**.
- [x] I've prefixed my Pull Request title is according to [gitmoji guide](https://gitmoji.carloscuesta.me/).

## Example

<details>
<summary>
:warning: Before merging please click here to expand and follow the guide.
</summary>
<br>

Please use `:twisted_rightwards_arrows:` at the beginning of your merge commit title.

Example 1:

<pre>
<code>
:twisted_rightwards_arrows: :bug: Fix Wrong Text Decoration at ...
</code>
</pre>

To get your commit message, just copy the first part of this pull request.

Example 2:
<pre>
<code>
**Changes:**

- Fixes Wrong Text Decoration at ...
- Fixes some typos
- ...

**Issues:**

Closes #NumberOfFixedIssue

PR: #NumberOfThisPR
</code>
</pre>
</details>

## Emojis

<details>
<summary>
Expand for a list of most used Emojis.
</summary>
<br>

Please prefix your commit messages with an Emoji.

Ref: https://gitmoji.carloscuesta.me/

| Description              | Glyphe               | Emoji  |
|--------------------------|----------------------|--------|
| Bugfix                   | `:bug:`              | 🐛     |
| Configuration releated   | `:wrench:`           | 🔧     |
| Cosmetic                 | `:lipstick:`         | 💄     |
| Dependencies Downgrade   | `:arrow_down:`       | ⬇️     |
| Dependencies Upgrade     | `:arrow_up:`         | ⬆️     |
| Formatting               | `:art:`              | 🎨     |
| Improving Performance    | `:zap:`              | ⚡️     |
| Initial commit           | `:tada:`             | 🎉     |
| Linter                   | `:rotating_light:`   | 🚨     |
| Miscellaneous            | `:package:`          | 📦     |
| New Feature              | `:sparkles:`         | ✨     |
| Refactoring Code         | `:recycle:`          | ♻️     |
| Releasing / Version tags | `:bookmark:`         | 🔖     |
| Removing Stuff           | `:fire:`             | 🔥     |
| Tests                    | `:white_check_mark:` | ✅     |
| Work In Progress (WIP)   | `:construction:`     | 🚧     |

</details>

### CODE REVIEW

**Comment by Vyperus** [comment]
I am not sure if the button is correctly placed since in "detail" the same button exists but in the navbar.

**Comment by alexanderkasten** [comment]
@Vyperus okay, but IMO a single button in the navbar looks a bit out of place 🤔 do you have a better idea? Or someone else?
@robpal94 @SteffenKn @cmg-dev @LeoTT

**Comment by Vyperus** [comment]
Well the button wouldn't be that alone since the navigation buttons are also up there.
Maybe add a little spacer to place the export button to the left from them with some margin.
Just like the "back"-button on the detail-view.

**Comment by robpal94** [comment]
I would add the button to the Navbar instead of the sidebar. 
In the `Design` view, this is the place where the user can find the button. Why should't he also find it there when navigating to the `Inspect` view?

File: aurelia_project/environments/prod.ts [hunk 1]
```diff
@@ -86,8 +86,13 @@ export default {
       navigateToInspect: 'processSolutionPanel:processEngineLinks:navigateToInspect',
       navigateToDesigner: 'processSolutionPanel:processEngineLinks:navigateToDesigner',
     },
-    inspectView: {
-      showInspectPanel: 'inspectView:inspectPanel:show',
+    inspect: {
+      shouldDisableTokenViewerButton: 'inspect:tokenViewerButton:disable',
+      exportDiagramAs: 'inspect:diagram:exportas',
```
Comment (robpal94) [inline aurelia_project/environments/prod.ts]:
```suggestion
      exportDiagramAs: 'inspect:diagram:exportAs',
```

File: aurelia_project/environments/stage.ts [hunk 2]
```diff
@@ -82,8 +82,13 @@ export default {
       navigateToInspect: 'processSolutionPanel:processEngineLinks:navigateToInspect',
       navigateToDesigner: 'processSolutionPanel:processEngineLinks:navigateToDesigner',
     },
-    inspectView: {
-      showInspectPanel: 'inspectView:inspectPanel:show',
+    inspect: {
+      shouldDisableTokenViewerButton: 'inspect:tokenViewerButton:disable',
+      exportDiagramAs: 'inspect:diagram:exportas',
```
Comment (robpal94) [inline aurelia_project/environments/stage.ts]:
```suggestion
      exportDiagramAs: 'inspect:diagram:exportAs',
```

**Comment by alexanderkasten** [comment]
> I would add the button to the Navbar instead of the sidebar.
> In the `Design` view, this is the place where the user can find the button. Why should't he also find it there when navigating to the `Inspect` view?
> 
> > okay, but IMO a single button in the navbar looks a bit out of place 🤔 do you have a better idea? Or someone else?
> 
> screenshot 2019-01-08 at 11 44 39
> 
> I count three navbar buttons when I'm in the Inspect view 😉
> 
> Also, I don't find it problematic to only display one Button. Better then moving the button with the same functionality to somewhere else imo.

Yes, @Vyperus said it before.

**Pull request merged**